<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbTitle">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>⌨️ Wizbz Keyboard Security</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lbEncryptedPlane">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>encrypted plane</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="ptEncryptedInput">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lbProcessList">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Process List</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leProcessSearch"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="lwProcess"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pbRefrsh">
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbAttach">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Attach</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbDetach">
             <property name="text">
              <string>Detach</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
